### PR TITLE
Fix/change front end submodule

### LINKED ISSUE #41: change frontend submodule

### PR DESCRIPTION
**Pull request by uier**
fix #41

### CODE REVIEW

File: .gitmodules
Comment (toto6038) [inline .gitmodules:6]:
```suggestion
	url = git@github.com:Normal-OJ/Sandbox.git
```

File: .gitmodules
Comment (toto6038) [inline .gitmodules:1]: 要不要都改用 SSH 的 URL？
git@github.com:Normal-OJ/Sandbox.git
git@github.com:Normal-OJ/Back-End.git

Comment (uier) [reply .gitmodules:1]: ok

**Review by Bogay — APPROVED** [review]

**Comment by uier** [comment]
thanks for the review, this PR relies on https://github.com/Normal-OJ/new-front-end/pull/152
Wait for its merge before merging.

**Pull request merged**
